--- original_Mjollnir/trunk/formula_lexer.mll	2010-06-21 15:34:08.000000000 +0200
+++ Mjollnir/trunk/formula_lexer.mll	2013-05-08 18:57:57.340960542 +0200
@@ -7,7 +7,7 @@
 let whitespace = ['\r' '\t' ' ']+
 
 rule normal_token = parse
-| ['A'-'Z']  [ 'a'-'z' 'A'-'Z' '0'-'9' ]* {
+| ['A'-'Z']  [ 'a'-'z' 'A'-'Z' '0'-'9' '_' ]* {
   match Lexing.lexeme lexbuf with
   | "True" -> Lex_true((), (cur_extent lexbuf))
   | "False" -> Lex_false((), (cur_extent lexbuf))
@@ -29,7 +29,7 @@
 
   | id -> Lex_u_identifier(id, (cur_extent lexbuf)) }
 
-| ['a'-'z']  [ 'a'-'z' 'A'-'Z' '0'-'9' ]* {
+| ['a'-'z' '_']  [ 'a'-'z' 'A'-'Z' '0'-'9' '_' ]* {
     Lex_l_identifier((Lexing.lexeme lexbuf), (cur_extent lexbuf)) }
 
 | "(" { Lex_lparen((), (cur_extent lexbuf)) }
